cfy_manager/main.py

```diff
@@ -543,45 +547,54 @@ def _print_finish_message(config_file=None):
 def _are_components_installed():
     return all(
         os.path.isfile(os.path.join(INITIAL_INSTALL_DIR, service_name))
-        for service_name in get_installed_services_names())
+        for service_name in config[SERVICES_TO_INSTALL])
 
 
 def _are_components_configured():
     return all(
         os.path.isfile(os.path.join(INITIAL_CONFIGURE_DIR, service_name))
-        for service_name in get_installed_services_names())
+        for service_name in config[SERVICES_TO_INSTALL])
```

Comment:
And shouldn't this be using configured service names (for which we might need /etc/cloudify/.configured/<service name>?

Reply:
I'll mark this as resolved and we can continue this discussion in that one thread.